lib/Interpreter/CIFactory.cpp

```diff
@@ -492,7 +492,7 @@ namespace {
       }
 
       static const char *CppInclQuery =
-        "echo | LC_ALL=C " LLVM_CXX " -xc++ -E -v - 2>&1 >/dev/null "
+        "echo | LC_ALL=C " LLVM_CXX " -xc++ -E -v /dev/null 2>&1 "
```

Comment:
It's hanging sometimes on OS X. But I see I missed the stdout redirect.
LC_ALL=C " LLVM_CXX " -xc++ -E -v /dev/null 2>&1 >/dev/null  ?